src/styles/index.css

```diff
@@ -18,6 +18,10 @@ html {
 
 svg.recharts-surface {
   overflow: visible;
+  position: absolute;
```

Comment:
`position: absolute` often causes issues down the line. E.g. if we want to add any element below the chart. margin/padding/position-relative is almost always a better solution for offsetting an element.